.markdownlint.yml

```diff
@@ -7,8 +7,7 @@
 default: true
 
 line-length:
-  # Line length checking is not strict by default.
-  strict: true
+  strict: false
```

Comment:
Please, revert this one and group links at the bottom of sections/pages and use:
```
<!-- markdownlint-disable line-length -->
  ...
<!-- markdownlint-enable line-length -->
```